R/sf.R

```diff
@@ -383,6 +383,10 @@ scale_type.sfc <- function(x) "identity"
 #' @usage NULL
 #' @format NULL
 CoordSf <- ggproto("CoordSf", CoordCartesian,
+  # parameters that control the parsing of labels plotted in the
+  # N or E directions
+  parse_N_labels = FALSE,
```

Comment:
Can we just have the user pass expressions as labels rather than making this an option?